.github/workflows/hawkscan-integration.yml

```diff
@@ -12,10 +12,11 @@ jobs:
           ref: ${{ github.event.client_payload.ref }}
       - name: Create app directory for ${{ github.event.client_payload.repoName }}
```

Comment:
I think you can skip this step.  When I was testing, the checkout created the apps directory under __tests__/integration-tests

Reply:
Going to leave this for now to keep the app and config directories, but maybe will change in the future